pkg/resource/plugin/plugin.go

```diff
@@ -55,6 +55,12 @@ func NewMissingError(info workspace.PluginInfo) error {
 }
 
 func (err *MissingError) Error() string {
+	if err.Info.Version != nil {
+		return fmt.Sprintf("no %[1]s plugin '%[2]s-v%[3]s' found in the workspace or on your $PATH, "+
```

Comment:
NIT: Do we use this syntax elsewhere vs just `%s`?

Reply:
I don't think so? I used it here so that I could refer to the same format argument twice in the same string.